alvr/client_core/src/connection.rs

```diff
@@ -1,17 +1,17 @@
 #![allow(clippy::if_same_then_else)]
 
 use crate::{
-    decoder::{self, DECODER_INIT_CONFIG},
+    decoder::{self, push_nal, DECODER_INIT_CONFIG},
```

Comment:
Prefer qualifying functions with one parent module, in the calling site.

Reply:
You mean calling it like `decoder::push_nal`, instead of importing?

Reply:
Yes

Reply:
this is resolved, btw
so I'm marking as so